pkg/common/utils.go

```diff
@@ -73,6 +73,12 @@ func GetEssentialTags(tagsInput interface{}, tagName string) (time.Time, int64,
 		for key, value := range typedTags {
 			tags = append(tags, MyTag{Key: key, Value: *value})
 		}
+	case []string:
+		for _, value := range typedTags {
+			if strings.Contains(value, "=") {
+				tags = append(tags, MyTag{Key: value[0:strings.Index(value, "=")], Value: value[strings.Index(value, "=")+1 : len(value)]})
```

Comment:
nit-pick: maybe using `strings.SplitN` would be clearer / simpler, CF: https://pkg.go.dev/strings#SplitN